axon-framework/deadlines/event-schedulers.md

```diff
@@ -8,7 +8,7 @@ To help understand this better lets take the scenario of a saga. It is easy to m
 
 In Axon, you can use an `EventScheduler` to schedule an event for publication. In the example of an invoice, you would expect the invoice to be paid within thirty days. A saga would, after sending the `CreateInvoiceCommand`, schedule an `InvoicePaymentDeadlineExpiredEvent` to be published in 30 days. The `EventScheduler` returns a `ScheduleToken` after scheduling an event. This token can be used to cancel the schedule, for example when a payment of an Invoice has been received.
 
-Axon provides two `EventScheduler` implementations: a pure Java one and one using Quartz 2 as a backing scheduling mechanism.
+Axon provides three `EventScheduler` implementations: a pure Java one, one using Quartz 2 as a backing scheduling mechanism and one using Axon Server as scheduler (from Axon 4.4 onwards).
```

Comment:
Oh, I've added this stuff to in [my PR](https://github.com/AxonIQ/reference-guide/pull/156) for framework :sweat_smile:

Reply:
reverted my changes to avoid conflicts.

Reply:
Gracias!